src/main/java/catan/services/util/play/PlayUtil.java

```diff
@@ -40,6 +40,12 @@ public void updateNextMove(GameBean game) throws GameException {
         }
     }
 
+    public void resetDices (GameBean game) {
```

Comment:
it is expected naturally, coz when the move is changed, old throw dice results became an invalid.  The same as preparationCycle and currentCycleBuildingNumber values became invalid after Preparation stage changes to Main and its values are nulling.